templates/tc467937/pod467937-new.yaml

```diff
@@ -17,7 +16,7 @@ spec:
       command: ["sh", "-c", "while true; do if [[ -e /etc/labels ]]; then cat /etc/labels; fi; if [[ -e /etc/annotations ]]; then cat /etc/annotations; fi; sleep 5; done"]
       volumeMounts:
         - name: podinfo
-          mountPath: /etc
```

Comment:
This is the failure reason.
Manual passed the fix and the checkpoint:
oc create -f pod467937-new.yaml
oc get po
kubernetes-metadata-volume-example   1/1     Running             0          10s
oc rsh kubernetes-metadata-volume-example
/ # ls -laR /data/podinfo-dir
/data/podinfo-dir:
total 0
drwxrwxrwt    3 root     root           120 Nov  6 09:31 .
drwxr-xr-x    3 root     root            25 Nov  6 09:31 ..
drwxr-xr-x    2 root     root            80 Nov  6 09:31 ..2019_11_06_09_31_18.756347822
lrwxrwxrwx    1 root     root            31 Nov  6 09:31 ..data -> ..2019_11_06_09_31_18.756347822
lrwxrwxrwx    1 root     root            18 Nov  6 09:31 annotations -> ..data/annotations
lrwxrwxrwx    1 root     root            13 Nov  6 09:31 labels -> ..data/labels

/data/podinfo-dir/..2019_11_06_09_31_18.756347822:
total 8
drwxr-xr-x    2 root     root            80 Nov  6 09:31 .
drwxrwxrwt    3 root     root           120 Nov  6 09:31 ..
-rw-r--r--    1 root     root           381 Nov  6 09:31 annotations
-rw-r--r--    1 root     root            58 Nov  6 09:31 labels